app/main.py

```diff
@@ -1,8 +1,48 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float, clean_power: int,
+                 average_rating: float, count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list) -> float:
+        total_income = 0.0
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                washing_price = \
+                    (
+                        car.comfort_class * (self.clean_power - car.clean_mark)
+                        * self.average_rating / self.distance_from_city_center
+                    )
+                total_income += washing_price
+                car.clean_mark = self.clean_power
+                self.wash_single_car(car)
```

Comment:
The line `car.clean_mark = self.clean_power` is redundant because the `wash_single_car` method, which is called immediately after, performs the same operation. You can remove this line to avoid redundancy.